app/controllers/profile_controller.rb

```diff
@@ -9,6 +11,30 @@ def show
     @user = current_user
   end
 
+  def change_password
+    passwd_form = params[:password_change]
```

Comment:
```suggestion
    password_form = params[:password_change]
```
and the password_form should be a model